config/config.go

```diff
@@ -15,8 +15,12 @@ import (
 	"github.com/qri-io/jsonschema"
 )
 
+// CurrentConfigVersion is the latest configuration number
+const CurrentConfigVersion = 1
```

Comment:
So this might be a minor quibble, but I could see an issue with calling this "version". These days, version tends to imply something like semver, and is expected to rarely change its major value, incrementing a minor number more often. Of course we're only using the term "version" internally, but it's still a visible term inside the textual config file. As an alternative, we could use "revision", which is more often used as a monotonically increasing value, like for example a build number.

I don't feel super strongly about this, just a thought.